docs/examples/general/data_loading/parallel_external_source.ipynb

```diff
@@ -14,12 +14,11 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            images_dir = "../../data/images/"
</pre><p>indentation seems to big here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Good catch, thanks.</p>